mne_bids/mne_bids.py

```diff
@@ -253,14 +254,12 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
     """
     # get MEASurement date from the data info
     meas_date = raw.info['meas_date']
-    if isinstance(meas_date, (np.ndarray, list)):
+    if isinstance(meas_date, tuple):
```

Comment:
can we not leave in `np.ndarray` and `list` and just add `tuple` ? Because this means we are forcing the user to use the bleeding edge `mne` ...

Reply:
yes, and then change once MNE 0.17 is out? would be fine with me :-)

Reply:
as long as it's not hard to support the old version of `mne` I'd do it ...

Reply:
I've seen users still using `0.15` :)